File: lib/models/estudio_medico.dart
class Estudio {
  int id;
  String estudio;
  String costo;


  Estudio(
    { 
    this.id,
    this.estudio,
    this.costo,
    });

  factory Estudio.fromJson(Map<String, dynamic>json){
    return Estudio(
      id: json['id'],
      estudio: json['estudio'],
      costo: json['costo'],
    );
  }
}

File: app/Models/estudio_medico.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Estudio_Medico extends Model
{
    protected $table = 'estudios_medicos';
    protected $fillable = [
        'estudio',
        'costo',
        'img'
    ];
    use HasFactory;
}
